src/index.html

```diff
@@ -17,16 +17,26 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form class="search-form search-form_big" data-qa="big">
+      <label class="search-form__label search-form__label_big">
+        <input
+          data-qa="keypress"
+          type="text"
+          placeholder="Try “Los Angeles”"
+          class="search-form__input search-form__input_big"
+        />
```

Comment:
The `label` element should have a `for` attribute which refers to the `id` of the `input` it labels. This improves accessibility as it allows screen readers to read out the label when the user is focused on the input. [HTML Content]